node/src/blockchain/blockchain_interface_utils.rs

```diff
@@ -0,0 +1,1412 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
```

Comment:
If you're establishing a brand new file, the tradition is to make the copyright up to date. 2024 in this case.